src/components/FilterBar/index.tsx

```diff
@@ -1,23 +1,43 @@
-import React from 'react';
-import DropdownFilter from '../DropdownFilter';
-import SearchFilter from '../SearchFilter';
-import { DropdownFilterConstant } from '../../constants'; 
-import './FilterBar.css';
+import React, { useEffect, useState } from "react";
+import DropdownFilter from "../DropdownFilter";
+import SearchFilter from "../SearchFilter";
+import { getFilters as getFiltersHelper } from "../../helpers/GetFilters";
+import "./FilterBar.css";
+import { DropDownData } from "../../interfaces";
 
 const FilterBar = () => {
-    return (
+  const [dropDownData, setDropDownData] = useState<DropDownData | null>(null);
+  const getData = async () => {
+    try {
+      const data = await getFiltersHelper();
+      setDropDownData((prevState) => {
+        return { ...prevState, ...data };
```

Comment:
You don't need to spread `prevState`, alwways have to override. Additionally this needs to be wrapped in `useCallback` and should be added as dep of `useEffect`